.travis.yml

```diff
@@ -5,7 +5,7 @@ language: python
 jobs:
   include:
     - stage: deploy
-      python: "3.6"
+      python: "3.7"
```

Comment:
As a RFE, this should probably be migrated to GitHub actions similarly to https://github.com/ome/omero-py/blob/cda0978e1041e1d4ef7927d7232a15eaf8fab2ed/.github/workflows/workflow.yml#L45-L62